Change console monitor path to /spice_auto.html

alextricity25 opened the pull request:
The way it's now, sending a request to the root path "/" will
respond with a 404. The proper base url to use is http://<host>:6082/spice_auto.html.
This commit changes the nova console monitor to send packets to this URL.

This commit also removes the HTTPS nova console monitor, as it's unused.

Connects https://github.com/rcbops/rpc-openstack/issues/1671

alextricity25 commented:
Running this code produced a healthy monitor:
![screenshot from 2017-01-13 10 56 18](https://cloud.githubusercontent.com/assets/4964964/21938879/dbeea884-d982-11e6-9c25-f52ce0dbcbb8.png)

alextricity25 commented:
FYI - This will need to be backported to the newton-14.0 branch before it's closed.

BjoernT commented:
@alextricity25  👎  What is the reason for this, we should never run TCP checks in gating

alextricity25 commented:
In my lab(fcfs-iad3-4), the http check didn't come up green for this monitor.
I just curl'ed http://internal_vip:6082, and it returns a "connection refused" message.
Thanks for bringing this up, this might not be a problem with the monitor configurations, but rather some network device blocking 6082.

devx commented:
I will have to echo @BjoernT 👎 

One should not use TCP unless we are just making sure that there exists 3 way hand shake.  Additionally if you are monitoring an HTTP service you should expect an OK 200.  So if port is not coming up it could indicate a change in the service, a dead server, bad network update, etc. Not to mention the case that we change the service to only respond to HTTPS traffic.

alextricity25 commented:
@BjoernT @devx I've updated the title and description of this fix to account for your feedback. The problem was that the monitor was sending heartbeats to http://URL:6082/ and responding with a 404 when the monitor was expecting a 200 OK. The proper base URL is http://URL:6082/spice_auto.html. This URL responds with a 200 OK. 
I've changed the monitor to send heartbeats to this URL and keep using HTTP.

BjoernT commented:
👍 Yes that is good. We internally fixed that monitor to use spice_auto.html, I guess that never made it back into this script :-(

alextricity25 commented:
recheck_swift
